content/source/docs/enterprise/before-installing/reference-architecture/gcp.md

```diff
@@ -265,12 +264,24 @@ primary GCP Region hosting the Terraform Enterprise application failing, the sec
 GCP Region will require some configuration before traffic is directed to
 it along with some global services such as DNS.
 
-- [Cloud SQL cross-region read replicas](https://cloud.google.com/sql/docs/postgres/replication/manage-replicas) can be used in a warm standby architecture or [Cloud SQL database backups](https://cloud.google.com/sql/docs/postgres/backup-recovery/restoring) can be used in a cold standby architecture.
+- [Cloud SQL cross-region read replicas](https://cloud.google.com/sql/docs/postgres/replication/cross-region-replicas)  can be used in a warm standby architecture. see also [Managing Cloud SQL read replicas](https://cloud.google.com/sql/docs/postgres/replication/manage-replicas) .
+
+  - Note that read replicas do not inherently provide high availability in the sense that there can be automatic failover from the primary to the read replica. As described in the above reference, the read replica will need to be promoted to a stand-alone Cloud SQL primary instance. Promoting a replica to a stand-alone Cloud SQL primary instance is an irreversible action, so when the failover needs to be reverted, the database must be restored to an original primary location (potentially by starting it as a read replica and promoting it), and the secondary read replica will need to be destroyed and re-established.
+
+  - GCP now offers a [high availability option for Cloud SQL](https://cloud.google.com/sql/docs/mysql/high-availability) databases which could be incorporated into a more automatic failover scenario.*
+
+- [Cloud SQL database backups](https://cloud.google.com/sql/docs/postgres/backup-recovery/restoring) can be used in a cold standby architecture.
+
+  - GCP now offers a [Point-in-time recovery](https://cloud.google.com/sql/docs/postgres/backup-recovery/pitr) option for Cloud SQL databases which could be incorporated into a backup and recovery scheme with reduced downtime and higher reliability.* 
```

Comment:
I get what you are saying here and was a little concerned for my wording. There are a couple of things in play here - we know customers have brought up these newer options so do want to head-off the impression we don't keep current so want to acknowledge them; and the new information will likely come out in a separate set of documents we are working on dedicated to backup/restore, at which time we will remove this topic from the RAs most likely. We can't wait to get other items corrected for that work to be done, so we want to get this published now.
I'm dong this - I have changed the note to reflect your wording and left the bullet items alone (so it is a common footnote for both). I've left "into this RA" even though it may be separate docs to avoid the ambiguity that it will be documented somewhere (it will be a link in the RA at least). Let me know if you don't like that handling.